com.vmware.horizon/com.vmware.horizon.yaml

```diff
@@ -0,0 +1,86 @@
+---
+app-id: com.vmware.horizon
+default-branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: vmware-view
+finish-args:
+  - "--share=ipc"
+  - "--device=dri"
+  - "--socket=x11"
+  - "--share=network"
+build-options:
+  env:
+    VERSION: 2106-8.3.0-18251983
+modules:
+  - name: mm-common
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: >-
+          https://download.gnome.org/sources/mm-common/1.0/mm-common-1.0.2.tar.xz
+        sha256: a2a99f3fa943cf662f189163ed39a2cfc19a428d906dd4f92b387d3659d1641d
+    cleanup:
+      - '*'
+  - name: sigc++-2
+    config-opts:
+      - '--disable-documentation'
+    sources:
+      - type: archive
+        url: >-
+          https://download.gnome.org/sources/libsigc++/2.10/libsigc++-2.10.6.tar.xz
+        sha256: dda176dc4681bda9d5a2ac1bc55273bdd381662b7a6d49e918267d13e8774e1b
+  - name: glibm
+    cleanup:
+      - /lib/glibmm-2.4
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/glibmm/2.64/glibmm-2.64.2.tar.xz
+        sha256: a75282e58d556d9b2bb44262b6f5fb76c824ac46a25a06f527108bec86b8d4ec
+  - name: atkmm
+    config-opts:
+      - '--disable-documentation'
+    cleanup:
+      - /lib/atkmm-1.6
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/atkmm/2.28/atkmm-2.28.0.tar.xz
+        sha256: 4c4cfc917fd42d3879ce997b463428d6982affa0fb660cafcc0bc2d9afcedd3a
+  - name: cairomm
+    config-opts:
+      - '--disable-documentation'
+    cleanup:
+      - /lib/cairomm-1.0
+    sources:
+      - type: archive
+        url: https://www.cairographics.org/releases/cairomm-1.12.2.tar.gz
+        sha256: 45c47fd4d0aa77464a75cdca011143fea3ef795c4753f6e860057da5fb8bd599
+  - name: pangomm
+    config-opts:
+      - '--disable-documentation'
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/pangomm/2.42/pangomm-2.42.1.tar.xz
+        sha256: 14bf04939930870d5cfa96860ed953ad2ce07c3fd8713add4a1bfe585589f40f
+  - name: gtkmm
+    config-opts:
+      - '--disable-documentation'
+    cleanup:
+      - /lib/gtkmm-3.0
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/gtkmm/3.24/gtkmm-3.24.2.tar.xz
+        sha256: 6d71091bcd1863133460d4188d04102810e9123de19706fb656b7bb915b4adc3
+  - shared-modules/libusb/libusb.json
+  - name: vmware-horizon-client
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://download3.vmware.com/software/view/viewclients/CART22FQ2/VMware-Horizon-Client-Linux-2106-8.3.0-18251983.tar.gz
+        sha256: 368d0b08c1d5be109ab70f692fe9809377f7cf7c20a46bbf72eedf73e59f1035
+    build-commands:
+      - tar xzvf x64/VMware-Horizon-Client-${VERSION}.x64.tar.gz
+      - cp -a VMware-Horizon-Client-${VERSION}.x64/lib "${FLATPAK_DEST}/lib"
```

Comment:
since this is x86_64 only you need to add a `flathub.json` to direct flathub to only build x86_64

https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64